.travis.yml

```
@@ -10,7 +10,7 @@ install:
   - composer install --no-interaction --prefer-source
 
 script:
-  - ./vendor/bin/phpcs
+  - ./vendor/bin/phpcs -n
```

We can ignore errors.

---

/warnings